src/Exceptions/InvalidCurrencyException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Exceptions;
+
+final class InvalidCurrencyException extends \LogicException
+{
+    public function __construct(string $currency)
+    {
+        parent::__construct(sprintf('Unknown format value: "%s"', $currency));
```

Comment:
No sure about the message as its not an `Unknown format` but an invalid value

Reply:
For the file SpecialBenefitType.php, I do not understand. The changes are well done and appear in the PR file.

<img width="1137" alt="Capture d’écran 2019-10-29 à 11 33 29" src="https://user-images.githubusercontent.com/5719739/67761020-bb611c80-fa42-11e9-8245-5097bc74aa7f.png">